playground/frontend/playground_components/lib/src/widgets/toasts/toast.dart

```diff
@@ -19,44 +19,52 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 
-import '../constants/sizes.dart';
+import '../../assets/assets.gen.dart';
+import '../../constants/colors.dart';
+import '../../constants/sizes.dart';
+import '../../models/toast.dart';
+import '../../models/toast_type.dart';
+import '../../playground_components.dart';
 
-const kNotificationBorderWidth = 4.0;
-const kMaxTextWidth = 300.0;
+const _borderWidth = 4.0;
+const _textWidth = 300.0;
 
-class BaseNotification extends StatelessWidget {
-  final String title;
-  final String notification;
-  final Color color;
-  final String asset;
+const _colors = UnmodifiableToastTypeMap(
+  error: BeamNotificationColors.error,
+  info: BeamNotificationColors.info,
+);
 
-  const BaseNotification({
-    super.key,
-    required this.title,
-    required this.notification,
-    required this.color,
-    required this.asset,
-  });
+final _iconNames = UnmodifiableToastTypeMap(
+  error: Assets.notificationIcons.error,
+  info: Assets.notificationIcons.info,
+);
+
+class ToastWidget extends StatelessWidget {
+  final Toast toast;
+
+  const ToastWidget(this.toast);
 
   @override
   Widget build(BuildContext context) {
-    return Stack(
-      children: [
-        _renderLeftBorder(context),
-        _renderNotificationContent(context),
-      ],
+    return Card(
+      child: Stack(
+        children: [
+          _leftBorder(context),
```

Comment:
May be change on stateless widgets?

Reply:
This is a moved code, let's keep it as it is.